controllers/user.controller.js

```diff
@@ -0,0 +1,118 @@
+const User = require("../models/user.model");
+const { extend } = require("lodash");
+const bcrypt = require("bcrypt");
+const jwt = require("jsonwebtoken");
+
+const getUsers = async (req, res) => {
+  try {
+    let users = await User.find({});
+    users = users.map((user) => {
+      user.password = undefined;
+      return user;
+    });
+    res.json({ success: true, users });
+  } catch (err) {
+    res.status(500).json({
+      success: false,
+      message: "Unable to get the list of users",
+      errMessage: err.message,
+    });
+  }
+};
+
+const findUser = async (req, res) => {
+  const { username, password } = req.body;
+  let user = await User.findOne({ username });
+  if (user) {
+    if (bcrypt.compareSync(password, user.password)) {
+      const token = jwt.sign({_id: user._id, name: user.name},process.env.JWT_SECRET, { expiresIn: "24h" });
+      res.json({ success: true, token });
```

Comment:
Great! :) 
What you can do is extract this into utils folder as a function like `signToken` or `generateTokenonSuccess`